### PR TITLE
Thermal Configuration for CIV with VM Manager

### PR DESCRIPTION
**Pull request by vilasrk**
This change will copy the customised configuration
for thermal and reload the service to get
the customised changes in effect.

Tracked-On: OAM-102072
Signed-off-by: Vilas R K <vilas.r.k@intel.com>

### CODE REVIEW

File: scripts/setup_host.sh [hunk 1]
```diff
@@ -543,6 +551,7 @@ ubu_update_fw
 install_vm_manager
 
 prepare_required_scripts
+start_thermal_daemon
```
Comment (YadongQi) [inline scripts/setup_host.sh:554]: is it really always needed? I think this is an optional case

Comment (vilasrk) [reply scripts/setup_host.sh:554]: thermal related epics will not work, if this change is not available.

**Review by YadongQi — CHANGES_REQUESTED** [review]
Add an argument to enable this feature

**Review by YadongQi — APPROVED** [review]

**Review by kaushlen — APPROVED** [review]

**Review by swaroopbalan — APPROVED** [review]

**Review by sgnanase — APPROVED** [review]

**Review by YadongQi — APPROVED** [review]

**Review by YadongQi — APPROVED** [review]

**Review by JaikrishnaNemallapudi — APPROVED** [review]
LGTM

**Pull request merged**